doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -474,6 +474,17 @@ arguments used for configuration.  Here is a table describing these emulated dev
           latter is ignored and the MAC address is set to the ``mac`` value.
           ``mac_seed`` will only be used when ``mac`` is not set.
 
+   * - ``virtio-gpu``
+     - Virtio GPU type device, parameters format is:
+       ``virtio-gpu,<geometry={widthxheight+x_off+y_off|fullscreen:screen_id},geometry={...}...]>``
```

Comment:
Yes, if "geometry" repeated, we will create multi-virtual display, just like qemu does. But for now, we only support 1 virtual display.

Reply:
Is there a meaningful error message that is displayed if or when the user tries to create multiple virtual displays?

Reply:
A few comments regarding line 479: 
1. If ACRN doesn't support multiple geometry parameters, I think we should remove the second one. 
2. It looks like line 479 has an extra closing bracket "]" at the end. 
3. If geometry is optional, put the whole param in brackets, such as: ``virtio-gpu[,geometry={<width>x<height>+<x_off>+<y_off> | fullscreen:<screen_id>}]``

Reply:
If user tries to create multiple virtual display, we only parse one virtual display and no error message now.

Reply:
> If user tries to create multiple virtual display, we only parse one virtual display and no error message now.

As @amyreye recommended, let's remove the second one then. Ideally we'd also print out a warning in case the user is trying to add a second virtual display (which he would likely not do if nothing in our documentation indicates he could ;-))